pkg/apis/pgv2.percona.com/v2/perconapgcluster_types.go

```diff
@@ -811,6 +811,10 @@ const (
 	// The value of the annotation will be a name of an existing backup job
 	AnnotationPGBackrestBackupJobName = AnnotationPGBackrestBackup + "-job-name"
 
+	// AnnotationPGBackrestBackupJobType is the annotation that is added to a PerconaPGClusterBackup.
+	// The value of the annotation will be a type of a backup (e.g. "manual" or "replica-create).
```

Comment:
Maybe we need to use `init-backup` or `stanza` backup instead of `replica-create` @egegunes WDYT?
`manual` -  I am not sure that it is correct in case of schedule backups, maybe we need to have  `manual`,  `schedule` and e.g. init